schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "clin_asser_trait_type",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "name",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "symbol",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "attribute_set",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "trait_relationship",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          },
+          {
+            "name": "id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "set_element_set_type",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "element_value",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "element_value",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "element_value",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          },
+          {
+            "name": "citation",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "xref",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "comment",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "type_attribute_set",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "attribute",
+            "datatype": "string"
+          },
+          {
+            "name": "citation",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "xref",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "comment",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "attribute",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "type_attribute",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_attribute",
```

Comment:
Do this and `attribute` have circular references? `attribute.type_attribute` and `type_attribute.attribute`